### PR TITLE
EE-676

### PR DESCRIPTION
**Pull request by garethgooch**

### CODE REVIEW

File: app/service/ClaimAmountService.scala [hunk 1]
```diff
@@ -78,4 +82,13 @@ class ClaimAmountService @Inject()(
         Seq(standardRate(claimAmount), scottishRate(claimAmount))
     }
   }
+
+  def filterRecords(taxCodeRecord: Seq[TaxCodeRecord]): Option[TaxCodeRecord] = {
+    taxCodeRecord.find(_.status == Live) match {
+      case Some(liveTaxCodeRecord) => Some(liveTaxCodeRecord)
+      case None if taxCodeRecord.nonEmpty => taxCodeRecord.headOption
+      case None if taxCodeRecord.isEmpty => None
+    }
```
Comment (vinnicombe) [inline app/service/ClaimAmountService.scala]: What happens if there are multiple records with status == Live?

File: test/service/ClaimAmountServiceSpec.scala [hunk 2]
```diff
@@ -108,6 +109,47 @@ class ClaimAmountServiceSpec extends SpecBase with MockitoSugar with ScalaFuture
...
+            starterRate = frontendAppConfig.taxPercentageScotlandBand1,
+            basicRate = frontendAppConfig.taxPercentageScotlandBand2,
+            higherRate = frontendAppConfig.taxPercentageScotlandBand3,
+            calculatedStarterRate = claimAmountService.calculateTax(frontendAppConfig.taxPercentageScotlandBand1, claimAmount),
+            calculatedBasicRate = claimAmountService.calculateTax(frontendAppConfig.taxPercentageScotlandBand2, claimAmount),
+            calculatedHigherRate = claimAmountService.calculateTax(frontendAppConfig.taxPercentageScotlandBand3, claimAmount)
+          ))
+      }
+
       "no tax code record must return both english and scottish rates" in {
```
Comment (vinnicombe) [inline test/service/ClaimAmountServiceSpec.scala]: Can we have a test that addresses the above comment and proves the behaviour?

File: app/service/ClaimAmountService.scala (hunk 1 above)
Comment (garethgooch) [reply app/service/ClaimAmountService.scala]: It will return the first record it has found that is live

Comment (vinnicombe) [reply app/service/ClaimAmountService.scala]: Or multiple Live with a mixture of Scottish and English tax codes?

**Pull request merged**